amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -155,8 +157,14 @@ export class SearchPage extends React.Component<SearchPageProps, SearchPageState
         key: ResourceType.table,
         content: this.getTabContent(this.props.tables, TABLE_RESOURCE_TITLE),
       },
-      // TODO PEOPLE - Add users tab
     ];
+    if (AppConfig.userFeature.enabled) {
+      tabConfig.push({
+        title: `Users (${ this.props.users.total_results })`,
+        key: ResourceType.user,
+        content: this.getTabContent(this.props.users, USER_RESOURCE_TITLE),
+      })
+    }
```

Comment:
I thought about how to do feature flags, but this is what I came up with... littering the codebase with if-statements. Anyone have other ideas or suggestions?

CC: @jornh @verdan

Reply:
I don’t have anything against those if-statements - I think there easy enough to read. (I just googled for javascript feature flags and a React related lib came up. Not sure if it adds enough value to add that extra dependency though.

One thing I would like to ask though is if the default for the user feature should be that it’s turned on? Speaking *for* that would be that the default Docker frontend can be used to demo the User/People feature with just a hard coded user (Roald Amundsen comes to mind) without requiring a local rebuild first. The other alternative of course is to implement #170

Reply:
That's a very good point about whether the default should be on or off. I do see value in having it enabled in the default Docker version. My assumption, and I may be totally wrong, would be that most companies would be launching Amundsen with tables only. Integration with users would come later, if at all. What I am trying to avoid is for a company to integrate just tables and have a bunch of broken links to missing profile pages.

#170 is an interesting solution, although there would be non-trivial work to invest there. Some of the configurations are really build configurations (enabling google analytics will change the processed root `index.html` file) while others could be runtime configs (like this one).